### PR TITLE
Add Threads icon

### LINKED ISSUE #8930: Request: Threads from Instagram
### Brand Name

Threads

### Website

https://threads.net

### Popularity Metric

Similarweb global rank: [Here](https://www.similarweb.com/website/threads.net/)
Similarweb global rank: [4](https://www.similarweb.com/website/instagram.com/)

### Official Resources for Icon and Color

svg: [Created from me](https://raw.githubusercontent.com/edizbaha/edizbaha/main/threads.svg)
color: `#000000`

### Additional Comments

Threads is Instagram's new app. Threads.net is not available for use at the moment. It will be available on July 6th.

[Play Store](https://play.google.com/store/apps/details?id=com.instagram.barcelona&gl=it)
[App Store](https://apps.apple.com/us/app/threads-an-instagram-app/id6446901002)

### Contributing

Yes

### PR DESCRIPTION
**Pull request by Saeris**
**Issue:** closes #8930

**Similarweb rank:** N/A
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

Threads is a new social media app by Instagram, made publicly available on July 5th, 2023. There appears to be no official brand guidelines page yet, so this SVG was pulled from the Wikipedia article for the app.

image

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -11975,6 +11975,11 @@
             "hex": "0099FF",
             "source": "https://www.threadless.com/about-us/"
         },
+        {
+            "title": "Threads",
+            "hex": "000000",
+            "source": "https://en.wikipedia.org/wiki/Threads_(app)#/media/File:Threads_(app)_logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:11981]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:Threads_(app)_logo.svg"
```

Please update the source to the Wikimedia Commons location 😄

File: icons/threads.svg
Comment (adamrusted) [inline icons/threads.svg:1]: Icon itself is a perfect match. If you can update the source as above, this'll be good to go.

**Review by adamrusted — CHANGES_REQUESTED** [review]
Please see comments regarding this PR.
Important to note that the `threads.net` site, which is the main domain for the app is currently sitting at a SimilarWeb rank of [2,875,528](https://www.similarweb.com/website/threads.net/) which would put it [squarely out of scope](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-an-icon) of this project at the moment. How do other @simple-icons/maintainers feel about this one?

**Comment by Saeris** [comment]
Yeah it's a brand new app so I'm not surprised that it has a low ranking at the time I filed this PR.

According to reports from Meta employees, Threads is already at 70m+ users as if this morning, so I expect that rank will change quickly.

I filed this PR because I want to use this icon in my social link sharing site, and figured it would be worth sharing in case others wanted to add links to their own sites. Can imagine that fairly soon many brands will be updating their social contacts to include their official account on Threads.

I'm not in a particular rush to get this merged, just thought I'd add some context as to why I submitted this in the first place.

**Comment by LitoMore** [comment]
The popular metric looks good to me. This product is well-known enough.

**Review by LitoMore — APPROVED** [review]

**Comment by adamrusted** [comment]
Alrighty, I'm happy to merge this one in, given I can see it's got over 10m downloads on android alone since it's launch!

**Pull request merged**